src/dbmate/build.py

```diff
@@ -0,0 +1,25 @@
+from ..buildlib import build_wheels
+
+NAME = 'dbmate'
+VERSION = '2.17.0'
+PYPI_VERSION = '2.17.0'
+SUMMARY = "A thin wrapper to distribute https://github.com/amacneil/dbmate via pip."
+LICENSE = "MIT"
+
+TARGET_TAG = {
+    'macos-arm64': 'macosx_11_0_arm64',
+    'linux-arm64': 'manylinux_2_17_aarch64.manylinux2014_aarch64.musllinux_1_1_aarch64',
+    'macos-amd64': 'macosx_10_9_x86_64',
+    'linux-amd64': 'manylinux_2_12_x86_64.manylinux2010_x86_64.musllinux_1_1_x86_64',
+}
+URL_TAG = {f"https://github.com/amacneil/dbmate/releases/download/{VERSION}/{NAME}-{target}": tag for target, tag in TARGET_TAG.items()}
```

Comment:
`https://github.com/amacneil/dbmate/releases/download/v2.17.0/dbmate-macos-amd64` is the URL, so I think a small tweak you need is `/v{VERSION}` (the `v` is needed here)